lib/util/makeNoMethodSetStateRule.js

```diff
@@ -44,6 +44,12 @@ function shouldBeNoop(context, methodName) {
     && !testReactVersion(context, '999.999.999'); // for when the version is not specified
 }
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ * @param {string} methodName Yay
+ * @param {(context: import('eslint').Rule.RuleContext) => boolean} [shouldCheckUnsafeCb] Yay
```

Comment:
"yay"?

Reply:
That's totally by accident, must have had the focus on the wrong window and accidentally wrote that

Reply:
Actually, now I remember, I added them when I tried to figure out what `valid-jsdoc` was complaining about – as it simply says "invalid" without saying what part of the jsdoc comment it considers to be invalid (eg. `eslint-plugin-jsdoc` [requires param descriptions](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/require-param-description.md#readme) in its non-ts oriented recommended config)